pkg/kubectl/cmd/cmd_test.go

```diff
@@ -25,9 +25,12 @@ import (
 	"net/http"
 	"os"
 	"reflect"
+	stdstrings "strings"
```

Comment:
The name `strings` is taken by `k8s.io/kubernetes/pkg/util/strings` which is also imported in this package.